### DISCUSSION TITLE
Manually uploading outcome data via TwoSampleMR

### DISCUSSION THREAD

**Discussion by alliyahharold** [MRCIEU/TwoSampleMR#460] [Q&A]
Hello,

I am wondering am attempting to manually upload my own outcome via R and want to know if the format listed [here](https://mrcieu.github.io/TwoSampleMR/articles/outcome.html#using-local-gwas-summary-data) is compulsory? We have our own exposure as well that matched the guidelines and was successfully run through MRbase.

Is it possible for us to use the same format as our exposure file? If not, can anyone give any insight to what is meant by "target_a1.outcome, target_a2.outcome, proxy_a1.outcome, proxy_a2.outcome, mr_keep.outcome" etc? Is there maybe an alternative outcome available? Thanks in advance for any resources or assistance!

**Comment by mightyphil2000** [comment]
If you wish to use your own outcome summary day , please follow the instructions here: https://mrcieu.github.io/TwoSampleMR/articles/outcome.html#using-local-gwas-summary-data

**Comment by mightyphil2000** [comment]
This question below was a little similar to your question, except they wanted to use local exposure and local outcome summary data https://github.com/MRCIEU/TwoSampleMR/issues/286

**Comment by laleoarrow** [comment]
"target_a1.outcome, target_a2.outcome, proxy_a1.outcome, proxy_a2.outcome, mr_keep.outcome" are columns when proxy snps are used in the tsmr analysis. They are not mandatory. You can add proxy snp for local outcome data if needed. My recent submitted PR is forked for this issue. #519
